src/components/ItemsList.js

```diff
@@ -73,6 +73,24 @@ const ItemsList = () => {
     history.push('/additem');
   };
 
+  const deleteItemFromShoppingList = (itemName) => {
+    const deleteItem = window.confirm(
+      `Are you sure you want to delete ${itemName} from shopping list?`,
+    );
+    if (deleteItem === false) return;
```

Comment:
This line works, but since deleteItem returns a boolean already, maybe using `if (!deleteItem)` might be more appropriate in this case.

Reply:
True, `if (!deleteItem)` seems to work as well.

Reply:
Thanks for the observation @Amaka202. The reasoning behind  using `deleteItem === false` is to make it obvious to the reader of the code  that `window.confirm` returns `true/false`. Using `!deleteItem` doesn't do that. `deleteItem` could be any `falsy` or `truthy`  value for that matter. It is to make the code easier to understand.

Reply:
@nibble0101 you make a good observation but I'll agree with @Amaka202 and @YuriDevAT here. In this situation, you can see that `deleteItem` can only be a Boolean because it is coming from `Window.confirm` right above it. If somebody isn't sure what `Window.confirm` returns, they can quickly check it's documentation (ex. https://developer.mozilla.org/en-US/docs/Web/API/Window/confirm) :slightly_smiling_face: .